packages/block-library/src/cover/index.php

```diff
@@ -43,9 +43,19 @@ function render_block_core_cover( $attributes, $content ) {
 			update_post_thumbnail_cache();
 		}
 		$current_featured_image = get_the_post_thumbnail_url();
-		$content                = preg_replace(
+
+		$styles = 'background-image:url(' . esc_url( $current_featured_image ) . '); ';
+
+		if ( isset( $attributes['minHeight'] ) ) {
+			$height_unit = $attributes['minHeightUnit'] ?? 'px';
+			$height		 = " min-height:{$attributes['minHeight']}{$height_unit}";
```

Comment:
Can't use the null coalescing operator. The minimum required PHP version for WP is still 5.6.

P.S. I think this will fail coding standard tests. Try running `composer lint packages/block-library/src/cover/index.php` in the project directory.

Reply:
I have updated the code.